src/index.css

```diff
@@ -120,17 +121,17 @@ ul {
   display: flex;
 } */
 
-.content {
+/* .content {
```

Comment:
There were multiple declarations for `.caption` and `.content`. Not sure if that was intentional or not